src/patternfly/components/ModalBox/modal-box.scss

```
@@ -37,6 +37,7 @@
 
   // Header main
   --#{$modal-box}__header-main--Gap: var(--pf-t--global--spacer--md);
+  --#{$modal-box}__header-main--PaddingBlockStart: var(--pf-t--global--spacer--sm);
```

wdyt about using `--pf-t--global--spacer--control--vertical--default` here? 

```suggestion
  --#{$modal-box}__header-main--PaddingBlockStart: var(--pf-t--global--spacer--control--vertical--default);
```